tests/features/CreatePostsTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+class CreatePostsTest extends FeaturesTestCase
+{
+    /**
+     * Creamos un posts.
+     */
+    public function test_a_user_create_a_post()
```

Comment:
__[code_sniffer]__
* Method name "CreatePostsTest::test_a_user_create_a_post" is not in camel caps format

<!-- Comment by [SideCI](https://sideci.com) -->